src/babel-plugin-transform-sx-tailwind/src/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -100,6 +100,39 @@ const __styles_3xIUAp = sx.create({
 });
 
 
+`;
+
+exports[`transform SX Tailwind conditional-operator.js: conditional-operator.js 1`] = `
+
+// @flow
+
+import type { Element } from 'react';
+
+export default function Example(isVisible: boolean): Element<'div'> {
+  return <div sxt={isVisible ? 'block' : 'hidden'}>Lorem lipsum</div>;
+}
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+// @flow
+import type { Element } from 'react';
+import sx from '@adeira/sx';
+export default function Example(isVisible: boolean): Element<'div'> {
+  return (
+    <div className={__styles_1spSrF(...(isVisible ? ['block'] : ['hidden']))}>Lorem lipsum</div>
```

Comment:
It would be nice to have a fixture with multiple names in the array as well. Something like `sxt={isVisible ? 'aa bb cc' : 'dd ee ff'}` -> `className={__styles_1spSrF(...(isVisible ? ['aa', 'bb', 'cc'] : ['dd', 'ee', 'ff']))}`